scripts/setup-project.sh

```diff
@@ -170,7 +170,7 @@ create_threadx_projects() {
       ;;
     rx65ncloudkit)
       echo Downloading Azure_RTOS_6...
-      ${FETCH} azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
+      ${FETCH} azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_ADU_Renesas_RX65N_Cloud_Kit_e2studio_gnurx_Sample_2022_04_28.zip
```

Comment:
Using an older version?

Reply:
Yes, older version has extra files (ADU content) which the basic sample does not build without, tested original fetch link but does not compile